Samples/AlgorithmsNvJpeg/Program.cs

```diff
@@ -0,0 +1,93 @@
+﻿using ILGPU;
```

Comment:
Question: is the copyright header missing?

Reply:
Added.